docs/user-guide/admin-cli.md

```diff
@@ -0,0 +1,141 @@
+vol        datastore1  Ubuntu_15.10   Sat Apr 16 20:13:18 2016  detached        N/A     100.00MB  14.00MB
+```
+
+Note that the `Policy` column shows the VSAN storage policy. Since these example volumes live on a
+VMFS datastore they do not have a storage policy and show up as `N/A'.
+
+### List selected columns
+
+Show only the selected columns.
+
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls -c volume,datastore,attached-to
+Volume     Datastore   Attached To VM
+---------  ----------  --------------
+large-vol  datastore1  detached
+vol        datastore1  detached
+```
+
+Note that the that the choices are given in a comma separated list with no spaces, and are shown in
+the help given above with `vmdkops_admin ls -h`.
+
+# policy
+
+Create, configure and show the VSAN policy names and their corresponding VSAN policies. Also show whether or not they are in use.
+
+### Help
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy -h
+usage: vmdkops_admin.py policy [-h] {rm,create,ls,update} ...
+
+positional arguments:
+  {rm,create,ls,update}
+    rm                  Remove a storage policy
+    create              Create a storage policy
+    ls                  List storage policies and volumes using those policies
+    update              Update the definition of a storage policy and allVSAN
+                        objects using that policy
+
+optional arguments:
+  -h, --help            show this help message and exit
+```
+
+### Create
+
+Create a VSAN storage policy.
+
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy create --name some-policy --content '(("proportionalCapacity" i0)("hostFailuresToTolerate" i0)'
+Successfully created policy: some-policy
+```
+
+Note that the VSAN storage policy given with `--content` is a standard VSAN storage policy. Please
+refer to the VSAN documentation for storage policy syntax.
+
+### List
+
+List all VSAN storage policies.
+
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy ls
+Policy Name  Policy Content                                             Active
+-----------  ---------------------------------------------------------  ------
+some-policy  (("proportionalCapacity" i0)("hostFailuresToTolerate" i0)  Unused
+```
+
+When creating a volume using `docker volume create`, the policy name should be given with the `-o`
+option such as `docker volume create --driver=vmdk --name=some-vol -o vsan-policy-name=some-policy`.
+The name of the volume will then show up in the `Active` column.
+
+### Update
+
+Update a VSAN storage policy.
+
+This command will update a VSAN storage policy for all volumes currently using this policy.
+
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy update --name some-policy --content '(("proportionalCapacity" i1)'
+This operation may take a while. Please be patient.
+Successfully updated policy: some-policy
```

Comment:
It's not currently in the code though, so not documented here. It does however show those stats if vsan volumes fail to update. 

```
return ('Successfully updated {0} volumes.\n'
           'Failed to update volumes: {0}').format(update_count, failed_updates)
```